community/modules/file-system/nfs-server/variables.tf

```diff
@@ -103,11 +103,16 @@ variable "scopes" {
 variable "local_mounts" {
   description = "Mountpoint for this NFS compute instance"
   type        = list(string)
+  default     = ["/tools", "/data"]
```

Comment:
nit: maybe a separate PR (or not at all). But would it make more sense to have a single mount as the default. Maybe just '/data'. Seems like having multiple mount points is an interesting feature but maybe not the desired default use case. 🤷 .

Reply:
I agree. I do also want to make sure we have a test that covers the multi-local-mount case, so I updated one of the test configs as well.